tools/update-npm.sh

```diff
@@ -40,11 +41,7 @@ rm -rf npm/
 
 echo "Copying new npm"
 
-tar zxf "$WORKSPACE"cli/release/npm-"$NPM_VERSION".tgz
-
-echo "Deleting temporary workspace"
-
-rm -rf "$WORKSPACE"
+tar zxf "$WORKSPACE"/cli/release/npm-"$NPM_VERSION".tgz
```

Comment:
I realize this comes from the previous implementation, but the use of the quotes seems… weird. I think we're supposed to put the whole argument inside one string.
```suggestion
tar zxf "${WORKSPACE}/cli/release/npm-${NPM_VERSION}.tgz"
```
or
```suggestion
tar zxf "$WORKSPACE/cli/release/npm-$NPM_VERSION.tgz"
```

Reply:
This is done in multiple places so I would not change it here. I think it's better to use another PR/commit to eventually make it consistent in the whole script.